src/edu/stanford/nlp/pipeline/StanfordCoreNLPClient.java

```diff
@@ -464,6 +465,8 @@ public void annotate(final Annotation annotation, final Consumer<Annotation> cal
           log.info("Trying to annotate locally...");
           StanfordCoreNLP corenlp = new StanfordCoreNLP(properties);
           corenlp.annotate(annotation);
+        } else {
+          annotation.set(CoreAnnotations.ExceptionAnnotation.class, t);
```

Comment:
Looks reasonable, although it looks like the ExceptionAnnotation didn't make it in to the code change

Reply:
Thanks!

Reply:
@AngledLuffa how do you mean it didn't make it into the code change?